clients/js/src/Collection.ts

```diff
@@ -471,24 +471,24 @@ export class Collection {
         query_embeddings,
         n_results,
         where,
-        query_text,
+        query_texts,
         where_document,
         include,
     }: {
         query_embeddings?: Embedding | Embeddings,
         n_results?: PositiveInteger,
         where?: Where,
-        query_text?: string | string[], // TODO: should be named query_texts to match python API
+        query_texts?: string | string[], // TODO: should be named query_texts to match python API
         where_document?: WhereDocument, // {"$contains":"search_string"}
         include?: IncludeEnum[] // ["metadata", "document"]
     }): Promise<QueryResponse> {
         if (n_results === undefined) n_results = 10
-        if (query_embeddings === undefined && query_text === undefined) {
+        if (query_embeddings === undefined && query_texts === undefined) {
             throw new Error(
-                "query_embeddings and query_text cannot both be undefined"
+                "query_embeddings and query_texts cannot both be undefined"
             );
-        } else if (query_embeddings === undefined && query_text !== undefined) {
-            const query_texts = toArray(query_text);
+        } else if (query_embeddings === undefined && query_texts !== undefined) {
+            const query_texts = toArray(query_texts);
```

Comment:
we need to rename this to say `const queryTextsArray`

Reply:
@suitedaces sorry - i think I was unclear, there is both `query_texts` which is used as an input, but also a new variable is defined inside the function called `query_texts_array/queryTextsArray` - we only wanted to rename the internal one, not all of them.

Reply:
I think I need a nap 🫠🔫 embarrassing